int8 startPt
int8 packagePt
int8 endPt
---
string navigationArray
